more/free-programming-cheatsheets.md

```diff
@@ -95,6 +95,7 @@
 ### <a id="cpp"></a>C++
 
 * [C++ Cheatsheet](https://www.codewithharry.com/blogpost/cpp-cheatsheet) - CodeWithHarry (HTML)
+* [C++ Cheatsheets & Infographics](https://hackingcpp.com/cpp/cheat_sheets.html) - Hacking C++ (HTML, PNG)
```

Comment:
```suggestion
* [Cheatsheets & Infographics](https://hackingcpp.com/cpp/cheat_sheets.html) - Hacking C++
```
Don't invent titles! Use the title given by the resource.

Reply:
Fixed this issue, Thanks for pointing out.